docs/modules/ROOT/assets/images/fire_riskpool_registration.md

```diff
@@ -0,0 +1,24 @@
+# Fire riskpool registration
+
+```mermaid
+sequenceDiagram
+    actor RK as RiskpoolKeeper
+    actor IO as InstanceOperator
+    participant G as GIF
+    participant R as FireRiskpool
+    participant RW as RiskpoolWallet
+    
+    
+    IO ->> G: grant role to RiskpoolKeeper
+    RK ->> R: deploy FireRiskpool contract
```

Comment:
Since deployment can happen independently before the rest of the actions, i would put this as the first interaction.